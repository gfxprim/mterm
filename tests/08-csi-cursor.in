10 10
A B C D E
\n\r
A\e[2\bC
B\e[2\bC
C\e[2\bC
D\e[2\bC
E
\n\r
A \e[\r2C
B \e[\r4C
C \e[\r6C
D \e[\r8C
E
\n\r
\e[20l
A \e[1\eA
B \e[1\eA
C \e[1\eA
D \e[1\eA
E
\n\r
