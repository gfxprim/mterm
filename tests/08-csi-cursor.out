 ----------
|A B C D E |
|A B C D E |
|A B C D E |
|A B C D E |
|          |
|          |
|          |
|          |
|          |
|          |
 ----------
size 10x10 cursor 4x0
